rakelib/default_plugins.rb

```diff
@@ -119,6 +119,8 @@ module RakeLib
       logstash-input-stdin
       logstash-input-tcp
       logstash-output-stdout
+      logstash-output-null
```

Comment:
we could yes, the idea is to use this in the agent pipeline for this test, so nothing is really output, but still everything is flowing for all the stages.